2022/21xxx/CVE-2022-21694.json

```diff
@@ -1,18 +1,93 @@
 {
```

Comment:
'https://github.com/onionshare/ onionshare/issues/1389' does not match '^(ftp|http)s?://\\S+$'